src/GeoJSONTables.jl

```diff
@@ -85,7 +85,7 @@ it should in some sense be defined.
 """
 function Base.getproperty(f::Feature, nm::Symbol)
     props = properties(f)
-    val = get(props, nm, missing)
+    val = nm in keys(props) ? getproperty(props, nm) : missing
```

Comment:
This seemed fine as is, looking at how getproperty calls get in JSON3?

https://github.com/quinnj/JSON3.jl/blob/cd857d0e3b10370311963c95fff7b5e07d47bd83/src/JSON3.jl#L98-L126

If this needs to be fully in terms of the property interface, perhaps it would need to be `nm in propertynames(props)`?

Reply:
Probably property names is the way to ho. Im actually not sure why the original doesn't work anymore either.

Reply:
Ok, must have been a red herring when I was debugging why `==` doesn't work now. `get` is fine.